events/memberUpdated.js

```diff
@@ -1,36 +1,51 @@
 const Discord = require('discord.js');
 
 function usernameUpdate(bot, oldMember, newMember) {
-  let userLogsChannel = bot.channels.find('name', 'user-logs');
+  const userLogsChannel = bot.channels.find('name', 'user-logs');
 
-  let oldMemberName, newMemberName;
+  // Create the embed
+  let embed = new Discord.RichEmbed();
+  embed.setColor(0xF1C40E);
+  embed.addField('User', newMember);
 
-  if (!oldMember.nickname) {
-    oldMemberName = oldMember.user.username;
-  } else {
-    oldMemberName = oldMember.nickname;
+  let embedDate = new Date(Date.now());
+  embed.setTimestamp(embedDate.toISOString());
```

Comment:
No! 😄

I mean the `toISOString` is required as discord fucking *dies* if you give it _anything else_ in that embed slot, but I don't know why I didn't just `Date.now().toISOString()` and skipped the date variable.